fsw/src/sc_cmds.c

```diff
@@ -538,6 +538,7 @@ void SC_ResetCountersCmd(const SC_ResetCountersCmd_t *Cmd)
 void SC_OneHzWakeupCmd(const SC_OneHzWakeupCmd_t *Cmd)
```

Comment:
I don't think "OneHz" as part of the function name or any associated documentation applies anymore?  Recommend checking for any other uses of 1hz/onehz or similar in case they are obsolete.

Reply:
I've removed 'OneHz' and similar phrases, and deprecated the current SC wakeup message ID, replacing it with a new one without the phrase